misc/python/materialize/cli/ci_logged_errors_detect.py

```diff
@@ -278,56 +287,98 @@ def annotate_logged_errors(log_files: list[str]) -> int:
     return len(unknown_errors)
 
 
-def get_error_logs(log_files: list[str]) -> list[ErrorLog]:
+def get_error_logs(log_file_names: list[str]) -> list[ErrorLog]:
     error_logs = []
-    for log_file in log_files:
-        with open(log_file, "r+") as f:
+    for log_file_name in log_file_names:
+        with open(log_file_name, "r+") as f:
             try:
-                data = mmap.mmap(f.fileno(), 0)
+                data: Any = mmap.mmap(f.fileno(), 0)
             except ValueError:
                 # empty file, ignore
                 continue
-            for match in ERROR_RE.finditer(data):
-                if IGNORE_RE.search(match.group(0)):
-                    continue
-                # environmentd segfaults during normal shutdown in coverage builds, see #20016
-                # Ignoring this in regular ways would still be quite spammy.
-                if (
-                    b"environmentd" in match.group(0)
-                    and b"segfault at" in match.group(0)
-                    and ui.env_is_truthy("CI_COVERAGE_ENABLED")
-                ):
-                    continue
-                error_logs.append(ErrorLog(match.group(0), log_file))
-            open_panics = {}
-            for line in iter(data.readline, b""):
-                line = line.rstrip(b"\n")
-                if match := PANIC_START_RE.match(line):
-                    service = match.group("service")
-                    assert (
-                        service not in open_panics
-                    ), f"Two panics of same service {service} interleaving: {line}"
-                    open_panics[service] = line
-                elif open_panics:
-                    if match := SERVICES_LOG_LINE_RE.match(line):
-                        # Handling every services.log line here, filter to
-                        # handle only the ones which are currently in a panic
-                        # handler:
-                        if panic_start := open_panics.get(match.group("service")):
-                            del open_panics[match.group("service")]
-                            if IGNORE_RE.search(match.group(0)):
-                                continue
-                            error_logs.append(
-                                ErrorLog(
-                                    panic_start + b" " + match.group("msg"), log_file
-                                )
-                            )
-            assert not open_panics, f"Panic log never finished: {open_panics}"
-
-    # TODO: Only report multiple errors once?
```

Comment:
Removed this TODO due to the introduction of `group_identical_errors` with https://github.com/MaterializeInc/materialize/pull/24400.